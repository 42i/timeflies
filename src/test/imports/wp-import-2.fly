import wp-import-2a.fly
import wp-import-2b.fly