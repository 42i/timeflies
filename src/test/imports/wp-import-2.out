Work package breakdown:
ALL
    pro; project of some sort
        aaa; part aaa
        bbb; part bbb
            xxx; detail xxx
            yyy; detail yyy
        ccc; part ccc
